### DISCUSSION TITLE
Suggestion: ~~multitrack~~ and multimodal stations, ~~custom lenght stations~~, road terminals, highways

### DISCUSSION THREAD

**Discussion by kiwinuken** [voxeltycoon/issues#375] [General]
It would significantly increase flexibility and organization if:

<s>multitrack stations could be created by merging stations placed next to each other, making trains choose the first stop spot available from a hierarchical signal system;</s>

multimodal stations could be created by merging stations from different modes - road, rail - to facilitate cargo transfer between modes;

<s>custom length stations could be created for trains, to have longer consists entirely inside stations;</s>

road terminals could be created with multiple road stops placed next to each other, to make it so that road vehicles don't have to wait in line for their turns to load/unload, instead going directly to the fist available stop spot - might need some changes to road vehicle behavior;

highways - road blocks that allow high speed in which stations cannot be built and at which side houses and other buildings cannot spawn, so that the player has a moderate amount of control over where cities grow and how they merge with each other.

**Comment by victorleblais** [comment]
You can already have station with up to 8 platforms and up to 250m (50 voxel blocks) in length.

I don't know if you suggest having higher limits?

**Comment by kiwinuken** [comment]
Really? I didn't find a way to do that as of yet... I've been plopping side by side single stations of the standard 5 blocks length all of this time!

I don't think 'normal' play would need stations with more than 8 platforms, but creative people might always find use for ludicrous amounts of anything, anyway. If the limit is built upon any performance or code issue, I guess it's ok, otherwise I guess not having a limit would be better. But that's not something that worries me, as I didn't even find the way to properly use the tool as its built =P

Multimodal stations, road terminals and highways still are suggestions. I'm gonna properly edit the header.
